code/__DEFINES/layers.dm

```diff
@@ -183,6 +183,8 @@
 #define SPLASHSCREEN_PLANE 9999
 #define SPLASHSCREEN_RENDER_TARGET "SPLASHSCREEN_PLANE"
 
+#define ABOVE_SPLASHSCREEN_LAYER 10000 //lol
```

Comment:
wouldn't that still display below splashscreen?

Reply:
Would it? I thought that a number higher means it will show up higher too.
![image](https://user-images.githubusercontent.com/54416597/170843747-76cb912d-63cf-4365-a764-4c7a8a4f6d83.png)
![image](https://user-images.githubusercontent.com/54416597/170843749-72ceff78-c6f6-499a-8310-c43663159cd1.png)
Hippie does it like this

Reply:
Well yes, but SPLASHSCREEN_PLANE is, well, a plane.
..okay, honestly, don't know how it works. I'm just assuming here that things display in order of planes and then in order of layers.

Reply:
I am pretty sure it's done via "priority" and that plane/layer is just a naming thing(?)
I need to test this in-game somehow.